scripts/update-lobster.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# this script pulls the evergreen-ci/lobster repo and copies the files so that
+# they can be served by evergreen
+# it should only be run from the base evergreen directory
+[[ -z "${LOBSTER_TEMP_DIR}" ]] && echo "missing environment variable LOBSTER_TEMP_DIR" && exit 1
+[[ -z "${EVGHOME}" ]] && echo "missing environment variable EVGHOME" && exit 1
+
+set -o errexit
+set -o xtrace
+
+echo $LOBSTER_TEMP_DIR
```

Comment:
I still have the question about whether we need to echo this. It is actually needed or is it just for debugging the script?

Reply:
oops, I meant to remove this debug log